apps/web/src/config/experminetalFeatures.ts

```diff
@@ -0,0 +1,25 @@
+export enum EXPERIMENTAL_FEATURES {
+  WebNotifications = 'web-notifications',
+}
+export type EnumValues<T> = T extends { [key: string]: infer U } ? U : never
+
+export type FeatureKeys = EnumValues<typeof EXPERIMENTAL_FEATURES>[]
+
+export const ctxKey = (key: EXPERIMENTAL_FEATURES) => `ctx-${key.toLowerCase()}`
```

Comment:
remember the naming convention. The name should be describing the purpose of the function

```suggestion
export const getCookieKey = (key: EXPERIMENTAL_FEATURES) => `ctx-${key.toLowerCase()}`
```